packages/common/src/content/compose.ts

```diff
@@ -556,6 +560,24 @@ interface ILayerViewDefinition extends ILayerDefinition {
 export const isLayerView = (layer: ILayerDefinition) =>
   (layer as ILayerViewDefinition).isView;
 
+const determineHubId = (
```

Comment:
I abstracted this out to a helper to prevent triple nested ternaries, but didn't feel that we need to expose this logic for others to consume. I left it in `compose.ts` because `getItemHubId` is a helper that is defined within this file.